scripts/release.sh

```diff
@@ -54,8 +55,16 @@ main() {
   fi
   RELEASE_VERSION="v${VERSION}"
   MINOR_VERSION=$(echo "${VERSION}" | cut -d. -f 1-2)
-  BRANCH=${BRANCH:-"release-${MINOR_VERSION}"}
-  REPOSITORY=${REPOSITORY:-"https://github.com/etcd-io/etcd.git"}
+
+
+  if [ "${CURRENT_BRANCH}" == 1 ]; then
+      # Trigger release in current branch, used in github workflow
+      REPOSITORY=$(pwd)
+      BRANCH=$(git branch)
```

Comment:
What do you expect `$(git branch)` to return? local git repository doesn't need to be on any branch.

Reply:
Note that we need to print the branch on which the release.sh run, 

```
log_callout "BRANCH=${BRANCH}"
```

Reply:
That's all for --current-branch mode

Reply:
FYI. in github workflow, the `$(git branch)` returns something like 

```
* (HEAD detached at pull/14860/merge)
```

Reply:
`git branch` will return list of all branches. On github workflow I expect it works because there is only one branch available. This code will break or even be misleading (may pick first branch from alphabetically sorted names) when running locally.

Reply:
Good catch: 
`git symbolic-ref --short HEAD` should work.

Reply:
Thanks for pointing out this. I need more sleep again :)

Reply:
Updated.